File: app/src/main/java/com/example/scb11/HomeActivity.java
package com.example.scb11;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class HomeActivity extends AppCompatActivity {
TextView homeTv;
EditText contactEt;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);
        homeTv = findViewById(R.id.tvHome);
        contactEt = findViewById(R.id.etContact);
//        String myName = getIntent().getExtras().getString("name");
//        homeTv.setText(myName);

    }

    public void getContact(View view) {
        String con = contactEt.getText().toString();
        homeTv.setText(con);
    }
}

File: app/src/main/java/com/example/scb11/HomeActivity.kt
package com.example.scb11

import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.EditText
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.example.scb11.network.MarsApi
import kotlinx.coroutines.GlobalScope
import kotlinx.coroutines.launch

class HomeActivity : AppCompatActivity() {
    var homeTv: TextView? = null
    var contactEt: EditText? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)
        homeTv = findViewById(R.id.tvHome)
        contactEt = findViewById(R.id.etContact)

        //        String myName = getIntent().getExtras().getString("name");
//        homeTv.setText(myName);
    }

    fun getContact(view: View?) {
//        String con = contactEt.getText().toString();
//        homeTv.setText(con);
        GlobalScope.launch {
            val listResult = MarsApi.retrofitService.getPhotos()
            Log.i("homeactivity",listResult.toString())

        }
    }
}